llvm/lib/TargetParser/AArch64TargetParser.cpp

```diff
@@ -134,17 +136,39 @@ std::optional<AArch64::CpuInfo> AArch64::parseCpu(StringRef Name) {
   return {};
 }
 
-void AArch64::PrintSupportedExtensions(StringMap<StringRef> DescMap) {
+void AArch64::PrintSupportedExtensions() {
   outs() << "All available -march extensions for AArch64\n\n"
          << "    " << left_justify("Name", 20)
-         << (DescMap.empty() ? "\n" : "Description\n");
+         << left_justify("Architecture Feature(s)", 55)
+         << "Description\n";
   for (const auto &Ext : Extensions) {
     // Extensions without a feature cannot be used with -march.
-    if (!Ext.Feature.empty()) {
-      std::string Description = DescMap[Ext.Name].str();
+    if (!Ext.UserVisibleName.empty() && !Ext.TargetFeature.empty()) {
```

Comment:
Which test in this PR checks this formatting? Just one row is fine.

Perhaps the CPU tests do but it seems like they only want FEAT_XYZ to show up.

Reply:
I've added a new unit test to cover the formatting in `TargetParserTest.cpp`.